nsot/models/attribute.py

```diff
@@ -128,6 +129,20 @@ def clean_resource_name(self, value):
 
     def clean_name(self, value):
         value = validators.validate_name(value)
+        resource_cls = self._get_model_cls()
+
+        # Grabbing concrete fields from the resource class to check if the
+        # attr name is not a concrete field on the resource model object
+        concrete_field_names = []
+        concrete_fields = resource_cls._meta.concrete_fields
+        for field in concrete_fields:
+            concrete_field_names.append(field.name)
```

Comment:
If refactored as my previous comments, this could just become:

```
resource_cls = self.get_resource_class()
concrete_field_names = resource_cls.get_concrete_field_names()
```

Reply:
Done.